translations/chi/pugsetup.phrases.txt

```diff
@@ -0,0 +1,490 @@
+"Phrases"
+{
+	"SetupMenuTitle"
+	{
+		"chi"		"设置选项"
+	}
+	"MapTypeOption"
+	{
+		"chi"		"地图类型"
+	}
+	"TeamTypeOption"
+	{
+		"chi"		"团队类型"
+	}
+	"TeamSizeOption"
+	{
+		"chi"		"团队人数"
+	}
+	"DemoOption"
+	{
+		"chi"		"录制比赛"
+	}
+	"AutoLiveOption"
+	{
+		"chi"		"自动录制"
+	}
+	"VetoMessage"
+	{
+		"chi"		"投票禁图将在几秒后开始···"
+	}
+	"VoteMessage"
+	{
+		"chi"		"投票选图将在几秒后开始···"
+	}
+	"AlreadyLive"
+	{
+		"chi"		"游戏已开始录制！"
+	}
+	"ReadyStatusPlayers"
+	{
+		"#format"	"{1:i},{2:i}"
```

Comment:
Please remove any `#format` lines - they should only be in the english translation file.